File: src/aot/benchmark.py
import time

def fib(n):
    if n < 2:
        return n
    return fib(n-1) + fib(n-2)

print("Starting Benchmark (Python Fib 40)...")
start = time.time()
res = fib(40)
end = time.time()
print(f"Result: {res}")
print(f"Time: {end - start}s")

File: src/aot/benchmark.c
#include <stdio.h>

long long fib(long long n) {
  if (n < 2)
    return n;
  return fib(n - 1) + fib(n - 2);
}

int main() {
  printf("Starting Benchmark (C O3 Fib 40)...\n");
  long long res = fib(40);
  printf("Result: %lld\n", res);
  return 0;
}
